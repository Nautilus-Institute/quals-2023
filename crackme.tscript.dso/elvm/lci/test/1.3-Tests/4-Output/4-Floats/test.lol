HAI 1.3
	VISIBLE 1.00 2.345 6.789
KTHXBYE
