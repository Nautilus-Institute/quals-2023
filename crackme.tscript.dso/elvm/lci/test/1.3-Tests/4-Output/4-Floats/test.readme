This test makes sure the print expression prints floating point decimals
correctly.
