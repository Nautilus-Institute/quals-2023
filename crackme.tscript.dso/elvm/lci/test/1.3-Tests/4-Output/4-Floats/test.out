1.002.346.78
